include/circt/Dialect/FIRRTL/FIRRTLStatements.td

```diff
@@ -131,15 +131,11 @@ def FPrintFOp : FIRRTLOp<"fprintf"> {
   }];
 
   let arguments = (ins ClockType:$clock, UInt1Type:$cond, StrAttr:$outputFile,
+                       Variadic<PrintfOperandType>:$outputFileSubstitutions,
                        StrAttr:$formatString, Variadic<PrintfOperandType>:$substitutions,
                        StrAttr:$name);
   let results = (outs);
-
-  let assemblyFormat = [{
-    $clock `,` $cond `,` $outputFile `,` $formatString ``
-    custom<FPrintfAttrs>(attr-dict) ` ` (`(` $substitutions^ `)`)? `:`
-    type($clock) `,` type($cond) (`,` qualified(type($substitutions))^)?
-  }];
+  let hasCustomAssemblyFormat = 1;
```

Comment:
Nit: does this _have_ to use a custom assembly format or is there some format that could work here? I realize that the double variadic may make this hard. I would've thought that the default format could handle this as long as there are appropriate anchors.

My motivation for using the default format is that it is just lower maintenance.